assets/charts/control-plane/calico/scripts/README.md

```diff
@@ -0,0 +1,30 @@
+# Calico Images
+
+This script downloads the calico images from docker hub and then uploads them to Quay.
+
+## Prerequisites
+
+- An AMD architecture based machine with docker installed.
+- An ARM architecture based machine with docker installed.
+
+## Run
+
+Execute the following steps once on AMD and then on ARM machine.
+
+### Docker login
+
+Run the following command in terminal:
+
+```bash
+docker login -u <username> quay.io
+```
+
+### Execute script
+
+Run the following command in terminal:
+
+```bash
+./calico-images.sh v3.18.1
```

Comment:
I think we can use variable here for version. What do you think @surajssd?

Reply:
I don't understand what do you mean? It is an example. With new calico release pass `v3.19.x` or whatever.

Reply:
NVM, not so important